src/snowflake/cli/plugins/nativeapp/codegen/snowpark/python_processor.py

```diff
@@ -220,15 +220,30 @@ def process(
 
                 # 4. Enrich entities by setting additional properties
                 for entity in collected_entities:
-                    _enrich_entity(
+                    can_proceed = sanitize_extension_function_data(
+                        ex_fn=entity, py_file=dest_file
+                    )
+                    if not can_proceed:
+                        cc.warning(
+                            f"Skipping generation of 'CREATE FUNCTION/PROCEDURE ...' SQL statement for this object."
+                        )
+                        continue
+                    enrich_entity(
                         entity=entity,
```

Comment:
nit: I fit it confusing that ex_fn and entity are both used for the same concept. Can we just keep ex_fn for clarity?